rllib/tests/test_dependency_tf.py

```diff
@@ -22,6 +22,10 @@
 
     config = (
         PPOConfig()
+        .api_stack(
```

Comment:
I thought the hybrid stack is off?

Reply:
I don't think we run this test anymore ...